packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -666,9 +688,9 @@ export function DownloadDetail({
       height="full"
       width="full"
     >
-      <Box style={{ maxWidth: 220, textAlign: 'center' }}>
+      <Box style={{ textAlign: 'center' }}>
```

Comment:
We should keep the maxWidth here; any reason we dropped it? New-line overflow is usually the better option

Reply:
Ah yeah.

On some languages the break of the text might look bad

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/039806bb-fcc3-4fdd-bca1-42b3377bab5b)
![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/7b093c54-501b-4672-8430-d0375fef4f74)